src/components/header/index.tsx

```diff
@@ -1,10 +1,11 @@
+import Logo from '../logoComponent/Logo';
 import ThemeSetting from '../themeSetting';
```

Comment:
this file has also been update by @Ank677  . please coordinate with him to prevent overriding each other's changes.